decidim-core/app/controllers/decidim/account_controller.rb

```diff
@@ -18,7 +18,7 @@ def update
       @account = form(AccountForm).from_params(params)
 
       UpdateAccount.call(current_user, @account) do
-        on(:ok) do |_account, unconfirmed_email|
+        on(:ok) do |unconfirmed_email|
```

Comment:
Since the email is not needed at all I am more comfortable keeping it as a boolean so I will rename it to `email_is_unconfirmed`.